apps/costcontrol/test/unit/settings_test.js

```diff
@@ -538,6 +538,44 @@ suite('Settings Test Suite >', function() {
         Settings.initialize();
       });
 
+      test('No corrections with invalid entries', function(done) {
```

Comment:
Please, add a regression test for the 0.1 value. I think we already have a test for these failing values.